brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/VIP/Coordinator.swift

```diff
@@ -127,6 +127,12 @@ class BaseCoordinator: NSObject,
         }
     }
     
+    func showDeleteKYCInfo() {
+        openModally(coordinator: DeleteKYCInfoCoordinator.self, scene: Scenes.DeleteKYCInfo) { vc in
+            vc?.prepareData()
```

Comment:
u only need to call prepareData, if u store smth in the dataStore, after which the tableView needs to be reloaded.

Reply:
updated. next pr.